vscode/src/ruby/asdf.ts

```diff
@@ -51,8 +51,19 @@ export class Asdf extends VersionManager {
   // file, but that may not be the case for a Homebrew installation, in which case the we'd have
   // `/opt/homebrew/opt/asdf/libexec/asdf.sh`, but the data directory might be `~/.asdf`
   async findAsdfDataDir(): Promise<vscode.Uri> {
+    // In order, the data locations are:
+    // 1. Default
```

Comment:
(nit) If we assign each possible path to variable, then we could avoid need for this comment.